presto-orc/src/main/java/com/facebook/presto/orc/QualifyingSet.java

```diff
@@ -47,6 +47,8 @@
     // numbers that include the nulls.
     boolean translateResultToParentRows;
 
+    boolean exceptionOnTruncate;
```

Comment:
I don't think this flag belongs to QualifyingSet. This class already has a lot of functionality and is very hard to understand. Let's not add complexity here.